### PR TITLE
Given feedback from end users, made a few updates to the readme

### PR DESCRIPTION
**Pull request by aaronlippold**

### CODE REVIEW

**Review by clintoncwolfe — CHANGES_REQUESTED** [review]
Hi Aaron,
  This still has some changes to the Terraform setup that I beleive are unrelated to your README changes, and it's hard to isolate the intent of the PR.  Could you trim it down a bit?  Thanks!

File: README.md [hunk 1]
```diff
@@ -63,6 +59,12 @@ control "aws-1" do
 end
 ```
 
+### Running your profile
+
+Then use `inspec exec my-profile` to execute your new profile.
+
+Our future intent is to support an `aws` target for InSpec/Train, so you may also pass credentials `inspec exec my-profile -t aws://accesskey:secret@region`.
```
Comment (clintoncwolfe) [inline README.md]: If you like, link to https://github.com/chef/train/issues/229

File: README.md [hunk 2]
```diff
@@ -80,10 +82,8 @@ end
  * `aws_iam_group`
  * `aws_iam_policy`
  * `aws_iam_role`
- ...
-
 
-## Tests
+## Development and Testing the AWS Resources Pack
```
Comment (clintoncwolfe) [inline README.md]: Developing and Testing the AWS Resources Pack

**Review by clintoncwolfe — CHANGES_REQUESTED** [review]
Two tiny changes, looks good overall - thanks!

File: README.md (hunk 1 above)
Comment (aaronlippold) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (aaronlippold) [reply README.md]: done

**Review by clintoncwolfe — APPROVED** [review]

**Pull request merged**